.ci/docs_versions.txt

```diff
@@ -0,0 +1 @@
+master
```

Comment:
Could we add other versions (v0.2.0 and v0.1.0) for a test? We could remove later if it's not necessary.

Reply:
Yeah, I'll do that once I get this to build correctly for just one version :)